actions/contract-tests/action.yml

```diff
@@ -45,7 +45,7 @@ runs:
     - name: 'Run Contract Tests'
       shell: bash
       run: |
-        curl ${{ inputs.token != '' && format('-H "Authorization: Token {0}"', inputs.token) || '' }} -s https://raw.githubusercontent.com/launchdarkly/${{ inputs.repo }}/${{ inputs.branch }}/downloader/run.sh | bash
```

Comment:
I would like to use `--retry-all-errors`, but the version of curl on my ubuntu is too old, so likely it would be too old in at least some CI. I think `--retry-connrefused` is safe (added in 2016).